proj1.py

```diff
@@ -0,0 +1,64 @@
+from stack import *
+from queue import *
+import sys
+
+def extractTags(line,queueOfTags): 
+    tag = ""
+    in_tag = False
+    
+    #looping through the lines in the file
```

Comment:
Is the loop looping through the lines in the file? Does extracttags care where you get the input from